app/controllers/integrations/base_controller.rb

```diff
@@ -105,12 +102,6 @@ def service_name
     @service_name ||= self.class.name.demodulize.sub('Controller', '').downcase
   end
 
-  def create_release_and_build_record
-    release = create_new_release
-    release = latest_release unless release && release.persisted?
-    create_build(release.version, [release])
```

Comment:
yes, good point ... the build would not be connected to the release ... which we don't need since the build should have the same sha as the release ... will fix for now and add a comment about it being unnecessary